apiserver/common/storagecommon/mock_test.go

```diff
@@ -41,6 +42,11 @@ func (s *fakeStorage) VolumeAttachment(m names.Tag, v names.VolumeTag) (state.Vo
 	return s.volumeAttachment(m, v)
 }
 
+func (s *fakeStorage) VolumeAttachmentPlan(host names.Tag, volume names.VolumeTag) (state.VolumeAttachmentPlan, error) {
```

Comment:
Is this method still being used? When I tried to look around I thought it was mostly changed to VolumeAttachmentPlans. Is there a reason we need both? I guess I'm a bit confused why one takes a host and a volume, and the other takes only the volume.

Reply:
Yes. See:

https://github.com/gabriel-samfira/juju/blob/add-storage-plugs/apiserver/facades/agent/storageprovisioner/storageprovisioner.go#L1127

One gets all attachments for a host. The other gets a particular attachment for one volume.